ui/packages/catalog-ui-search/src/main/webapp/component/metacard-preview/metacard-preview.view.js

```diff
@@ -22,22 +22,18 @@ const store = require('../../js/store.js')
 const user = require('../singletons/user-instance.js')
 const preferences = user.get('user').get('preferences')
 const wreqr = require('../../js/wreqr.js')
+import React from 'react'
+import { renderToString } from 'react-dom/server'
 
 function getSrc(previewHtml, textColor) {
-  return (
-    '<html class="is-iframe is-preview" style="font-size: ' +
-    preferences.get('fontSize') +
-    'px; color: ' +
-    textColor +
-    ';">' +
-    '<link href="css/styles.' +
-    document
-      .querySelector('link[href*="css/styles."]')
-      .href.split('css/styles.')[1] +
-    '" rel="stylesheet">' +
-    previewHtml +
-    document.querySelector('[data-theme]').cloneNode(true).outerHTML +
-    '</html>'
+  const fontSize = preferences.get('fontSize')
+
+  return renderToString(
+    <html
+      dangerouslySetInnerHTML={{ __html: previewHtml }}
```

Comment:
The html that goes here is already escaped - https://github.com/codice/ddf/blob/master/catalog/transformer/catalog-transformer-preview/src/main/java/org/codice/ddf/transformer/preview/PreviewMetacardTransformer.java#L40

Reply:
`dangerouslySetInnerHTML` is a bad name

Reply:
I agree with @rymach, approval pending changes.

Reply:
Ok guys I'll email zuckerberg and give him your feedback on that method name.